content/reference/sql/saved-queries.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Saved Queries
+---
+
+# Saved Queries
+
+Dolt is a database designed for sharing. Dolt ships with a feature called "Saved Queries". Saved queries are queries that are created and versioned along with the database itself. You can use saved queries to make your Dolt database easier to use by adding examples of how the data in the database could be queried.
+
+To create a saved query, you can use the Dolt CLI to save a SQL query for future use by yourself or others. This query is written to a special Dolt system table `dolt_query_catalog`. You can add, modify, or delete saved queries by changing that system table.
+
+```
+us-businesses $ dolt sql --save "Example saved query" -q "show tables"
++----------------+
+| Table          |
++----------------+
+| business_types |
+| businesses     |
+| naics          |
+| sic            |
++----------------+
+us-businesses $ dolt sql -q "select * from dolt_query_catalog"
++---------------------+---------------+---------------------+-------------+-------------+
+| id                  | display_order | name                | query       | description |
++---------------------+---------------+---------------------+-------------+-------------+
+| Example saved query | 1             | Example saved query | show tables |             |
++---------------------+---------------+---------------------+-------------+-------------+
+us-businesses $ dolt sql -q "delete from dolt_query_catalog where id ='Example saved query'"
```

Comment:
It would be cool to add an example somewhere in here of running the named query with `dolt sql -x "Example saved query"` just so people know about that way to use them from the command line, too.

Reply:
Great idea. I'll break it into two code blocks. One create and execute. The other modify.